media-plugins/gst-plugins-qt5/gst-plugins-qt5-1.18.4.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GST_ORG_MODULE=gst-plugins-good
+
+inherit gstreamer-meson
+
+DESCRIPTION="Qt5 QML video sink plugin for GStreamer"
+KEYWORDS="~amd64"
+IUSE="+egl orc wayland +X"
```

Comment:
orc should be default enabled, BUT qt plugins don't use orc, so it shouldn't be in the IUSE at all. The QA warning is a false positive from a wrong check in new eclasses [that's yet unsolved](https://bugs.gentoo.org/810417)